### ISSUE TITLE
pip download doesn't work

### ISSUE THREAD

**Issue by Mahm-MoodForFood** [BenTimor/Trading212API#1]

**Comment by BenTimor** [comment]
I know. I had to remove it.
I'll upload it again very soon.

**Comment by Elsafti** [comment]
Hi, thanks a lot for the module. I also could not pip it and downloaded the files manually on Python 3.8.5. I run it and it hangs on after:
trading = CFD("my email", "mypassword")

So I get the time out exception with an empty message:

TimeoutException                          Traceback (most recent call last)
<ipython-input-33-a27e2f709c23> in <module>
...
TimeoutException: Message: 


I would really appreciate your help. Thanks again and have a wonderful Christmas time!

Best regards
Hisham

**Comment by Mahm-MoodForFood** [comment]
Hey, 

I have had some issues with this program. I have started to fix it and will have a merge request ready once it is done. However no promises on when it will be done. 

Merry Christmas 
Mahm-MoodForFood

**Comment by BenTimor** [comment]
I started to notice that more and more people are using this API. I'll fix it and upload it to PIP very soon.

**Comment by BenTimor** [comment]
The API is fixed and uploaded to [PIP](https://pypi.org/project/Trading212/) again. If you still get any timeouts, Open a new issue and I'll check it.
